### PR TITLE
update scripts

### PR DESCRIPTION
**Pull request by nini-faroux**

### CODE REVIEW

File: dao/app/Main.hs
Comment (Randy1Burrell) [inline dao/app/Main.hs]: Let's remove this

**Review by Randy1Burrell — DISMISSED** [review]
@nini-faroux great work

**Comment by nini-faroux** [comment]
Thanks @Randy1Burrell  - just cleaned it up a bit more there and set it ready for review again

File: dao/dao.cabal
Comment (Randy1Burrell) [inline dao/dao.cabal]: Can we change this to "2024 Yaadlabs Ltd."?

**Review by Randy1Burrell — DISMISSED** [review]

**Review by Randy1Burrell — DISMISSED** [review]
LGTM!

File: dao/dao.cabal
Comment (nini-faroux) [reply dao/dao.cabal]: Sure, I'll do that in a moment

**Comment by nini-faroux** [comment]
Thanks @Randy1Burrell  - the copyright is updated now, so just needs another approval before merging

**Review by Randy1Burrell — DISMISSED** [review]
LGTM!

**Review by Randy1Burrell — DISMISSED** [review]

**Comment by nini-faroux** [comment]
@Randy1Burrell Oh not sure what that 'dismissed stale review' is about - when you get a chance to approve it again feel free to merge it yourself then.

**Pull request merged**
